### ISSUE TITLE
Link variable-hint element to dropdown button element

### ISSUE THREAD

**Issue by LucyGwilliamAdmin** [open-sdg/open-sdg#859]
**Page 78 in DAC report**

**Issue from DAC report:**
The hint text describing the dependency is not programmatically associated with the field.

**Solution from DAC report:**
Use aria-describedby attributes to programmatically associate instructions with dropdown fields

**Example from DAC report:**

```
<div class="variable-selector disallowed child" data-field="Region"
  data-has-data="true">
    <button class="accessBtn" tabindex="0" aria-expanded="false" aria-describedby="variable-hint">
        <h5>Region<i class="fa fa-chevron-down"></i></h5>
    </button>
    <div class="bar">…</div>
    <div class="variable-options">…</div>
    <div class="variable-hint" id="variable-hint">Available when 'Country' is selected</div>
    <div class="no-data-hint">No options available because of filters above</div>
</div>
```

**Comment by jwestw** [comment]
Hi I have created a feature branch for this and I see in [indicator 1.4.2](http://sdgdev-813006012.eu-west-1.elb.amazonaws.com/jw-variable-hint/1-4-2/) that aria-described-by text is present.  
`<button class="accessBtn" tabindex="0" aria-expanded="true" aria-described-by="variable-hint-Type of Documentation">
        <h5>Type of Documentation<i class="fa fa-chevron-down"></i></h5>
      </button>`
I am not sure if this means the fix is successful. I need some more explanation on this one. @brockfanning  please ping me on Slack. Thanks
